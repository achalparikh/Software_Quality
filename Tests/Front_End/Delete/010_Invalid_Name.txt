login
admin
delete
Jeff Doe
00012
logout
quit