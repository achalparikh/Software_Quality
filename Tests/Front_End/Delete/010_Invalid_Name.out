Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as an admin
Enter a command:
Enter an account holder:
Error, user Jeff Doe does not match any accounts
Enter a command:
Error, invalid command, enter a command:
You are now logged out
Login to start:
